lib/heex_formatter/formatter.ex

```diff
@@ -343,17 +371,36 @@ defmodule HeexFormatter.Phases.Format do
   #
   #   should close the tag in the next line.
   #   <p class="some-class">  \nShould break line
-  defp tag_contains_line_break?(html, tag) do
-    last_opened_tag = ~r/<\s*#{tag}[^>]*.(?!.*<\s*#{tag}[^>]*>(.*?).+).*/s
-
-    last_opened_tag
-    |> Regex.run(html)
-    |> case do
-      nil ->
-        false
+  defp tag_open_with_line_break?(buffer, tag) do
```

Comment:
Now we don't need regex anymore since we have a list of binaries within a list. So all we need to do here is to go until this tag has been open and check if it contains a line break. I guess it will be faster than before 🤔